foruma-frontend/src/stories/Page.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <article>
+    <my-header
+      :user="user"
+      @login="$emit('login')"
+      @logout="$emit('logout')"
+      @createAccount="$emit('createAccount')"
+    />
+
+    <section>
+      <h2>Pages in Storybook</h2>
+      <p>
+        We recommend building UIs with a
+        <a href="https://componentdriven.org" target="_blank" rel="noopener noreferrer">
```

Comment:
Codacy has a fix for the issue: [Replace `·href="https://componentdriven.org"·target="_blank"·rel="noopener·noreferrer"` with `⏎··········href="https://componentdriven.org"⏎··········target="_blank"⏎··········rel="noopener·noreferrer"⏎········`](https://app.codacy.com/gh/agabani/foruma/pullRequest?prid=7388752)
```suggestion
        <a
```